bin/pycbc_make_skymap

```diff
@@ -143,6 +143,7 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, ifar, ifos, thresh_SNR,
         command.append("--spin1z"),command.append(str(spin1z))
         command.append("--spin2z"),command.append(str(spin2z))
         command.append("--low-frequency-cutoff"),command.append(str(f_low))
+        command.append("--high-frequency-cutoff"),command.append(str(f_upper))
```

Comment:
Does this work if you don't provide the option? I.E. it will try to do str(None) and probably raise an error, won't it?

Reply:
if this is the case, this should probably be in a if statement so that it is really optional.

Reply:
+1